vis_services/tests/test_internals.py

```diff
@@ -11,7 +11,16 @@
 from vis_services.lib import author_network
 from vis_services.lib import paper_network
 from vis_services.lib import tf_idf
+import logging
```

Comment:
It looks like you tore out all the logging statements in these tests after you were done. If so, it'd probably be worth removing the logger definitions here.